packages/moonstone/ExpandableItem/ExpandableItem.js

```diff
@@ -250,6 +250,15 @@ const ExpandableItemBase = kind({
 		 */
 		open: PropTypes.bool,
 
+		/**
+		 * Prevents rendering the transition container on mount.
```

Comment:
Independent of the prop naming comment, the "on mount" portion isn't really accurate in this context.